hledger/Hledger/Cli/Commands/Register.hs

```diff
@@ -49,6 +49,7 @@ registermode = hledgerCommandMode
     ("fuzzy search for one recent posting with description closest to "++arg)
   ,flagNone ["related","r"] (setboolopt "related") "show postings' siblings instead"
   ,flagNone ["invert"] (setboolopt "invert") "display all amounts with reversed sign"
+  ,flagReq  ["sort"] (\s opts -> Right $ setopt "sort" s opts) "FlagHelp" "Help"
```

Comment:
More help to come I expect. Also once things settle we'll want to mention it in Register.md.

Reply:
Whoops, can't believe I forgot to update that...will fix along with adding documentation!